docs/defining-data-models.md

```diff
@@ -0,0 +1,126 @@
+In this section, we explain how to define `data models`, an important component of the OEF Search & Discovery. It allows agents to describe themselves and to discover the services/resources they are interested in.
+
+In a sentence, a `DataModel` is a set of `attributes`, and a `Description` of a service/resource is an assignment of those attributes.
```

Comment:
same as above for _attributes_

Reply:
maybe `DataModel` and `Description` work better in code format than italic, so I'd leave them.